module3/web/src/main/resources/application.properties

```diff
@@ -0,0 +1 @@
+spring.profiles.active = default
```

Comment:
This is not required. Spring Boot should set active profiles to `[default]` by default.